### PR TITLE
Remove iOS copy of CSSLayout, update podspec

### PR DESCRIPTION
**Pull request by javache**
We're checking in a central copy of css-layout under ReactCommon. @astreet already made the xcodeproj point at it, now we can actually remove the code copy (pending internal changes).

I've also updated the Podspec to deal with the new location and fixed the xcodeproj and Podspec to handle <CSSLayout/CSSLayout> includes correctly.

### CODE REVIEW

**Comment by javache** [comment]
cc @alloy, would you mind checking the Podfile? It works alright when I test it locally through `:path` but fails when I run `pod lib lint`

File: React.podspec [hunk 1]
```diff
@@ -25,15 +25,22 @@ Pod::Spec.new do |s|
   s.source              = { :git => "https://github.com/facebook/react-native.git", :tag => "v#{s.version}" }
   s.default_subspec     = 'Core'
   s.requires_arc        = true
-  s.platform            = :ios, "7.0"
```
Comment (ericvicenti) [inline React.podspec]: Does this end support for iOS 7? Should this be documented somewhere?

Comment (javache) [inline React.podspec]: We ended support for iOS7 a while back, we just didn't update the podspec.

**Comment by javache** [comment]
Landing internally.

**Pull request closed without merging**